README.md

```diff
@@ -76,40 +76,44 @@ Put your SSL certificate as `./volumes/web/cert/cert.pem` and the private key th
 no password as `./volumes/web/cert/key-no-password.pem`. If you don't have
 them you may generate a self-signed SSL certificate.
 
-## Starting/Stopping
+### Starting/Stopping
 
-### Start
+#### Start
 ```
 docker-compose start
 ```
 
-### Stop
+#### Stop
 ```
 docker-compose stop
 ```
 
-### Update
+### Removing D
 
-Make sure to backup Mattermost data before proceeding.
+#### Remove the containers
 ```
-docker-compose down
-git pull
-docker-compose build
-docker-compose up -d
+docker-compose stop && docker-compose rm
 ```
 
-## Removing
-
-### Remove the containers
+#### Remove the data and settings of your Mattermost instance
 ```
-docker-compose stop && docker-compose rm
+sudo rm -rf volumes
 ```
 
-### Remove the data and settings of your Mattermost instance
+## Update Docker to Latest Version
```

Comment:
I suggest to rephrase to "Update Mattermost to latest version" as this section explain how to upgrade Mattermost and not Docker.